common/psbt_keypath.c

```diff
@@ -14,20 +13,30 @@ void psbt_set_keypath(u32 index, const struct ext_key *ext, struct wally_map *ma
 	u32 path[1];
 	path[0] = index;
 
-	if (wally_map_keypath_add(map_in,
-				       ext->pub_key, sizeof(ext->pub_key),
-				       fingerprint, sizeof(fingerprint),
-				       path, 1) != WALLY_OK)
-		abort();
+	if (is_taproot) {
+		if (wally_psbt_output_taproot_keypath_add(output,
+												 ext->pub_key + 1, sizeof(ext->pub_key) - 1,
+												 NULL, 0,
+												 fingerprint, sizeof(fingerprint),
+												 path, 1) != WALLY_OK)
```

Comment:
Guessing your local tabs are 4 spaces?  Because this is very indented!

Reply:
ok im in libwally mode; what's the expected indent level :)